modules/SofaMiscForceField/src/SofaMiscForceField/MeshMatrixMass.inl

```diff
@@ -1230,8 +1223,8 @@ void MeshMatrixMass<DataTypes, MassType>::computeMass()
     clear();
 
     // prepare to store info in the vertex array
-    helper::vector<MassType>& my_vertexMassInfo = *d_vertexMassInfo.beginEdit();
-    helper::vector<MassType>& my_edgeMassInfo = *d_edgeMassInfo.beginEdit();
+    helper::vector<MassType>& my_vertexMassInfo = *d_vertexMass.beginEdit();
```

Comment:
Use WriteOnlyAccessor() instead of the dangerous beginEdit()

Reply:
this is already done in PR #2193 
This one is just a Ctrl + shift + H